tests/Particle.ts

```diff
@@ -83,8 +84,9 @@ describe('Particle', () => {
                 expect(testParticle.particle.fill).to.eql(squareShapeOptions.particles.shape.options.square.fill);
             });
 
-            xit('should set shapeData to the configured shape data matching the chosen shape whenever multiple shapes are specified for container Particles', () => {
+            it('should set shapeData to the configured shape data matching the chosen shape whenever multiple shapes are specified for container Particles', () => {
                 testContainer.reset(multipleShapeTypeOptions);
+                testContainer.addShapeDrawer('image', new ImageDrawer());
```

Comment:
Same as the comment in TestContainer.ts, this test should work without this line for changes in a recent commit